### PR TITLE
Ascension.AscensionLauncher version 1.79

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60504)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Package requires dependencies:
image

**Review by ItzLevvie — DISMISSED** [review]

**Comment by solomoncyj** [comment]
Can anyone help me do the tank file?

**Review by ItzLevvie — DISMISSED** [review]

**Comment by solomoncyj** [comment]
.

File: manifests/a/Ascension/AscensionLauncher/1.79/Ascension.AscensionLauncher.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ascension.AscensionLauncher
+PackageVersion: 1.79
+PackageLocale: en-US
+Publisher: Ascension
```
Comment (ItzLevvie) [inline manifests/a/Ascension/AscensionLauncher/1.79/Ascension.AscensionLauncher.locale.en-US.yaml:7]:
```suggestion
Publisher: Project Ascension
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**